src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -83,6 +83,7 @@
     <PackageReference Include="System.Reactive.Linq" Version="5.0.0" />
     <PackageReference Include="System.Reactive.Core" Version="5.0.0" />
     <PackageReference Include="System.Runtime.Loader" Version="4.3.0" />
+    <PackageReference Include="System.Text.Json" Version="6.0.9" />
```

Comment:
Can you add a comment with a note here? Something like:

```xml
<!--
Explicitly referencing System.Text.Json to get two bug fixes for the Source Generator. Without this, the 6.0.0 version 
of the Source Generator is used, which causes issues. This reference can be removed in future framework versions.

(and then add links to those two issues?) 
-->
```

Reply:
Added!